wiki/Article_Style_Guide/en.md

```diff
@@ -1,192 +1,369 @@
-Article Style Guide
-===================
+# Article Style Guide
 
-**Outdated**
+The Article Style Guide serves as a way to help keep consistency in clarity, formatting, and layout between all articles of the osu!wiki.
 
-This article aims to provide a comprehensive style guide for all osu!wiki articles to follow. Although Wikipedia already provides a more general [style guide](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style), a more specific one is necessary for *osu!* specific guidelines. As such, only guidelines pertaining to the osu!wiki and its basic formatting rules should be included here.
+All articles should use plain English.
+Your word choice should explain such things in layman's terms (basically ask yourself, "If the reader is reading this, would they have to lookup any of the words?").
 
-Notability
-----------
+All articles must have proper grammar, correct spelling, and correct information.
+Know that reviewers may ask for changes in your pull request for blunders or suggestions.
+A good osu!wiki writer should read these reviews to help improve the overall quality of these articles to ensure for the best for an osu!wiki reader.
```

Comment:
L10 : Revise. Suggested :  ``A good osu!wiki writer should read these reviews to help improve the overall quality of these articles to ensure the most optimal experience for an osu!wiki reader``.